github-workflows-kt/docker-compose.yml

```diff
@@ -22,7 +22,10 @@ services:
   jaeger:
     image: jaegertracing/all-in-one:latest
     networks:
-      - reverse_proxy
+      reverse_proxy:
+        # An abstract host name, used by the application. Allows abstracting
+        # out what component is the receiver of traces.
+        aliases: [traces_collector]
```

Comment:
No idea how this is supposed to work with reverse_proxy.

If this does what I think it does, 'http://trace_collector:16686' should work inside other containers.
By default only 'http://jaeger:16686' would be published.

Is this what we want to change?

Reply:
https://github.com/jaegertracing/jaeger/blob/4d1c7d8ddc23e5492772b3f985205dd423fbd363/docker-compose/monitor/prometheus.yml#L9

which seems to be exactly what your example is doing in the end

Reply:
Yes, that's exactly the goal :) Basically an alias for `jaeger`.